lib-android/src/test/java/com/devinci/lib/view/SignInButtonTest.java

```diff
@@ -0,0 +1,102 @@
+package com.devinci.lib.view;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.support.v4.content.res.ResourcesCompat;
+import android.util.AttributeSet;
+import com.devinci.lib.BuildConfig;
+import com.devinci.lib.R;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.res.Attribute;
+import org.robolectric.res.ResourceLoader;
+import org.robolectric.shadows.RoboAttributeSet;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.robolectric.Shadows.shadowOf;
+
+@RunWith(RobolectricGradleTestRunner.class) public class SignInButtonTest {
+
+  private Context context;
+  private Drawable expectedBackground;
+  private Drawable expectedForeground;
+
+  @Before public void setUp() throws Exception {
```

Comment:
Throws in those methods can be omitted - applies to `@Before`, `@After` and `@Test` methods (and added back if there really are exceptions that can be thrown).